types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -1113,3 +1172,10 @@ export interface LiveState<T> {
 }
 
 export function suspenseSentinel(): never;
+
+/**
+ * Context that will be provided to live resolvers if
+ * `resolverContext` is set on the Relay Store.
+ * This context will be passed as the third argument to the live resolver
```

Comment:
This comment is true, but kinda misleading in the context of types. The resolver function will not be invoked with this specific type, but rather with the type specified in the relay config. Can you please improve the documentation to reflect that, and perhaps link to docs on the config?